src/sentry/quotas/base.py

```diff
@@ -657,11 +693,24 @@ def check_accept_monitor_checkin(self, project_id: int, monitor_slug: str):
 
         return PermitCheckInStatus.ACCEPT
 
+    def check_accept_checkin(self, data_category: DataCategory, project_id: int, slug: str):
```

Comment:
Are the `checkin` and `slug` methods necessary?

Reply:
We don't need this for uptime. Check-ins are not sent by customers but produced by us. So if a uptime monitor is disable we'll just stop producing checks.